pkg/cluster/internal/create/actions/config/config.go

```diff
@@ -19,6 +19,8 @@ package config
 
 import (
 	"bytes"
+	"fmt"
+	"net"
```

Comment:
this whole commit can be dropped but I'm not fighting with git at 2am and it's not going to show up in the git history anyhow.